File: Geeks4Geeks/Misc/Replace_0_with_5.cpp
// { Driver Code Starts
#include <bits/stdc++.h>
using namespace std;

int convertFive(int n);

// Driver program to test above function
int main() {
    int T;
    cin >> T;
    while (T--) {
        int n;
        cin >> n;
        cout << convertFive(n) << endl;
    }
}// } Driver Code Ends
/*you are required to complete this method*/
int convertFive(int n) {
    // Your code here
    int i = 0,answer = n;
    while(n > 0)
    {
        int temp = n%10;
        if(temp == 0)
        {
            answer += 5 * pow(10,i);
        }
        i++;
        n = n/10;
    }
    return answer;
}

File: Geeks4Geeks/G4G-Solutions/replace_0_with_5.py
# https://practice.geeksforgeeks.org/problems/replace-all-0s-with-5/1

def convertFive(n):
    l = str(n).replace('0','5')
    return l



#{ 
#  Driver Code Starts
# Your code goes here
if __name__=='__main__':
    t = int(input())
    for i in range(t):
        print (convertFive(int(input().strip())))
# } Driver Code Ends
